dask/system.py

```diff
@@ -34,20 +60,12 @@ def cpu_count():
 
     # Check cgroups if available
     if sys.platform == "linux":
-        # The directory name isn't standardized across linux distros, check both
-        for dirname in ["cpuacct,cpu", "cpu,cpuacct"]:
-            try:
-                with open("/sys/fs/cgroup/%s/cpu.cfs_quota_us" % dirname) as f:
-                    quota = int(f.read())
-                with open("/sys/fs/cgroup/%s/cpu.cfs_period_us" % dirname) as f:
-                    period = int(f.read())
-                # We round up on fractional CPUs
-                cgroups_count = math.ceil(quota / period)
-                if cgroups_count > 0:
-                    count = min(count, cgroups_count)
-                break
-            except Exception:
-                pass
+        for quota, period in _enumerate_cgroup_cpu_quota():
+            # We round up on fractional CPUs
+            cgroups_count = math.ceil(quota / period)
+            if cgroups_count > 0:
+                count = min(count, cgroups_count)
+            break
```

Comment:
It looks like we're only ever taking one result from `_enumerate_cgroup_cpu_quota`. I'm wondering if we should simplify the code by changing `_enumerate_cgroup_cpu_quota` into a function that only ever returns one result or process multiple quotas (e.g., if it were possible for v1 and v2 to co-exist).

Reply:
Thank you for your feedback @hendrikmakait!

I did a little bit of digging (read the manual of cgroup) and found this:

> Although cgroups v2 is intended as a replacement for cgroups v1, the older system continues to exist (and for compatibility reasons is unlikely to be removed).  Currently, cgroups v2 implements only a subset of the controllers available in cgroups v1.  The two systems are implemented so that both v1 controllers and v2 controllers can be mounted on the same system.  Thus, for example, it is possible to use those controllers that are supported under version 2, while also using version 1 controllers where  version 2  does  not  yet  support those controllers.  The only restriction here is that a controller can't be simultaneously employed in both a cgroups v1 hierarchy and in the cgroups v2 hierarchy.

So if read this right it means you can't find the CPU quota in both variants at the same time, even if v1 and v2 can co-exist due to the controller restriction.

Meaning that it would make sense to simplify `_enumerate_cgroup_cpu_quota` a bit. I'll get right on making the necessary adjustments. Thanks!